ml-agents/mlagents/trainers/lesson_controller.py

```diff
@@ -0,0 +1,122 @@
+import os
```

Comment:
Didn't realize this was (mostly) migrated code. Can curriculium.py be deleted?

Reply:
Even though the ideal scenario is to replace curriculum with some of this code, we don't want to do this rightaway. Having said that, lot of the code logic (and code) is portable here.